### PR TITLE
0x API price clarification and allowance guide

### PR DESCRIPTION
**Pull request by fragosti**
- Clarifies what `price` means. 
- Creates a new setting allowance guide. 
- Refers to allowance guide and fixes some small issues.

Screen Shot 2020-02-06 at 1 49 03 PM
Screen Shot 2020-02-06 at 1 49 16 PM
Screen Shot 2020-02-06 at 1 49 22 PM

### CODE REVIEW

File: mdx/guides/how-to-set-your-token-allowances.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# How to set your token allowances
+
+Some interactions with 0x require or are improved by setting token allowances,
+or in other words, giving 0x's smart contracts permission to move certain tokens on your behalf. 
+Depending on the context, you will have to provide an allowance to one of the [AssetProxy](/docs/guides/v3-specification#assetproxy) contracts
+or [StakingProxy](/docs/guides/v3-staking-specification#architecture) contract. 
+You can look up the addresses for our smart contract deployments in the [0x cheat sheet](/docs/guides/0x-cheat-sheet#mainnet-1).
+
+You can read more about [the ERC20 token allowance function](https://tokenallowance.io/), but this guide will focus on how to set them.
```
Comment (dave4506) [inline mdx/guides/how-to-set-your-token-allowances.mdx]: Should we add a quick blurb for which contracts to approve for basic situations or ways to understand when or why you would approve the staking proxy, etc.

File: mdx/guides/how-to-set-your-token-allowances.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+## Setting Allowances on Etherscan
+
+There are a lot of dApps that let you set your allowances in Metamask (including [RadarRelay](https://app.radarrelay.com/)).
+Here we will use Etherscan as they allow you to interact with any smart contract.
```
Comment (dave4506) [inline mdx/guides/how-to-set-your-token-allowances.mdx]: Statement felt little confusing, it is correct but had to take a double take to understand it.

**Review by dave4506 — COMMENTED** [review]
Very minor nits but nice to see this exist now!

File: mdx/guides/how-to-set-your-token-allowances.mdx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+You can look up the addresses for our smart contract deployments in the [0x cheat sheet](/docs/guides/0x-cheat-sheet#mainnet-1).
+
+You can read more about [the ERC20 token allowance function](https://tokenallowance.io/), but this guide will focus on how to set them.
+
+
+## Setting Allowances on Etherscan
+
+There are a lot of dApps that let you set your allowances in Metamask (including [RadarRelay](https://app.radarrelay.com/)).
+Here we will use Etherscan as they allow you to interact with any smart contract. 
+To set your WETH allowance for the ERC20AssetProxy contract you can navigate to the [Dapp view for the WETH ERC20](https://etherscan.io/dapp/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2#writeContract) contract.
```
Comment (xianny) [inline mdx/guides/how-to-set-your-token-allowances.mdx]: Nit: `the 0x ERC20Proxy contract` or `the 0x ERC20 Asset Proxy`

File: mdx/guides/how-to-set-your-token-allowances.mdx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+Here you can use Metamask to call the `approve` method to approve the ERC20AssetProxy for the max uint256 amount which is `115792089237316195423570985008687907853269984665640564039457584007913129639935`.
+
+![Setting an approval in Etherscan!](/images/docs/approval.png)
```
Comment (xianny) [inline mdx/guides/how-to-set-your-token-allowances.mdx:17]: Can we include an explanation:
> Provide the address of the 0x ERC20AssetProxy contract. The updated address can be found in @0x/contract-addresses https://github.com/0xProject/0x-monorepo/blob/development/packages/contract-addresses/addresses.json#L5

**Review by xianny — CHANGES_REQUESTED** [review]
Yay, thanks for doing this. Looks great!

Nit: The guide refers to `ERC20AssetProxy` contract in a few places, but the name of the contract is `ERC20Proxy` in the node packages and contract files. Can we call it `0x ERC20Proxy smart contract` or `0x ERC20 asset proxy smart contract` in the guides? Or whatever you think sounds good, I just think we should stay away from using `ERC20AssetProxy` as a name because it could be confusing.

File: mdx/guides/how-to-set-your-token-allowances.mdx (hunk 1 above)
Comment (fragosti) [reply mdx/guides/how-to-set-your-token-allowances.mdx]: I just went for a rephrase here.

**Review by xianny — APPROVED** [review]

**Pull request merged**